zookeeper-server/src/main/java/org/apache/zookeeper/Version.java

```diff
@@ -41,6 +42,8 @@ public static String getBuildDate() {
         return BUILD_DATE;
     }
 
+    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE",
+            justification = "If SNAPSHOT is missing from version number this is redundant")
```

Comment:
Let's say "QUALIFIER" instead of "SNAPSHOT".

Reply:
Rephased the message